deepmd/train/trainer.py

```diff
@@ -412,14 +427,16 @@ def _init_session(self):
                 log.info('receive global variables from task#0')
             run_sess(self.sess, bcast_op)
 
-    def train (self, train_data, valid_data=None) :
+    def train (self, train_data = None, valid_data=None) :
 
         # if valid_data is None:  # no validation set specified.
         #     valid_data = train_data  # using training set as validation set.
 
         stop_batch = self.stop_batch
         self._init_session()
-
+        if self.is_compress:
+            self.saver.save (self.sess, os.getcwd() + "/" + self.save_ckpt)
+            return
```

Comment:
`train` should not be called. 
we may provide a new method like `save_compressed`